lib/cocoapods/installer/xcode/pods_project_generator/target_installer.rb

```diff
@@ -17,6 +17,11 @@ class TargetInstaller
           #
           attr_reader :target
 
+          # @return [Boolean] use_new_build_system
+          #         Wether we're currently using the new build system
+          #
+          attr_accessor :use_new_build_system
```

Comment:
please make this an `attr_reader`

Reply:
Can also set it as `false` to default so you do not have to change all the tests